### PR TITLE
feat(new relic solutions): created new documentment for the transitio…

### PR DESCRIPTION
**Pull request by homelessbirds**
…n to the new UI

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Review by paperclypse — APPROVED** [review]
It's looking good to me!

**Review by akristen — DISMISSED** [review]
GREAT doc, and with such a short turn around!

I made some light language copy edits. You hit all the important stuff someone would be confused about / want to know immediately, so no changes or clarification needed on that end. Everything is captured + made sense. 

Things I loved:

- Collapsers + their titles. This is a clever way to involve all our possible personas without cluttering up the doc! Also makes it f-scannable too, if you aren't at that point of implementing whatever feature/capability that section is about. I left an @ for Rob if you want to tinker with the indents within the collapsers. This is such a small nitpick tho that I doubt anyone will notice it / be bothered by it.
- The structure of the page is good too. You chose great H2s :) 
- Good tone. Friendly, casual but isn't fluffy or vague

My only ~critique is to maybe work the intro sentence a bit better. I think what you want to convey is good, but it reads a little clumsy. Maybe something snappier, 'We've simplified our UI to make it customizable for you.' ? Idk. Also fine to just leave it as it is because I don't predict anyone not using the UI because of what's there. 

Also, just made some light language edits that I want to stress are totally optional. I get wanting to push something out / getting nervous about changes that interrupt the site build! Transition guides are temporary so what's here is great for that purpose :)

File: src/content/docs/new-relic-solutions/new-relic-one/introduction-new-relic-one.mdx [hunk 1]
```diff
@@ -25,7 +25,7 @@ import nr1QueryBuilder from 'images/NR1-query-builder.png'
 
 import nr1Dashboards1 from 'images/NR1-dashboards-1.png'
 
-New Relic provides an integrated platform where you can explore core capabilities like querying data and building charts, dig deeper into more curated [observability experiences](/docs/full-stack-observability/monitor-everything/get-started-new-relic-monitoring-tools/get-started-full-stack-observability#h2-start-anywhere) in the UI, and use our [alerting and applied intelligence](/docs/alerts-applied-intelligence) tools. With the New Relic platform, you can see and act on all the data throughout your entire ecosystem.
+and building charts, dig deeper into more curated [observability experiences](/docs/full-stack-observability/monitor-everything/get-started-new-relic-monitoring-tools/get-started-full-stack-observability#h2-start-anywhere) in the UI, and use our [alerting and applied intelligence](/docs/alerts-applied-intelligence) tools. With the New Relic platform, you can see and act on all the data throughout your entire ecosystem.
```
[the remark this answers is not in the document]
Comment (homelessbirds) [reply src/content/docs/new-relic-solutions/new-relic-one/introduction-new-relic-one.mdx:28]: Good catch! Total mistake

File: src/content/docs/new-relic-solutions/new-relic-one/new-navigation-transition-guide.mdx [hunk 2]
```diff
@@ -0,0 +1,194 @@
+---
+title: New navigation UI transition guide
+tags:
+  - New Relic 
+  - Use New Relic
+  - Get started
+metaDescription: We've updated the UI. Learn about where everything is now.
+---
+
+import home from 'images/transition-home.png'
```
[the remark this answers is not in the document]
Comment (homelessbirds) [reply src/content/docs/new-relic-solutions/new-relic-one/new-navigation-transition-guide.mdx:10]: I'll do a follow up:)

**Pull request merged**